src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fieldSet">
+        <legend>Personal information</legend>
+
+        <label class="formField" for="surname">
+          Surname:
+          <input
+          type="text"
+          autocomplete="off"
+          name="surname"
+          id="surname"
+          required
+          >
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item, format your attributes correctly everywhere
```suggestion
          <input
            type="text"
            autocomplete="off"
            name="surname"
            id="surname"
            required
          >
```